tests/integration/targets/aci_access_port_to_interface_policy_leaf_profile/tasks/main.yml

```diff
@@ -65,6 +65,29 @@
     type: fex
     leaf_interface_profile: fexintprftest
 
+- name: Add a new fex policy group to the fexintprftest fex profile
```

Comment:
you may want to add a task to set var so that you don't need to set host/username/pasword... in every task

Reply:
@xinyuezhao Added set var task to initialize basic parameters.